package.json

```diff
@@ -1,17 +1,24 @@
 {
-  "name": "@datadog/datadog-react-native-wizard",
-  "version": "0.0.1",
+  "name": "datadog-react-native-wizard",
```

Comment:
what is the reason for the name change?

Reply:
for `npx` to work, the `name` has to have the same value as the `bin`. 
And the `bin` does not accept `@datadog/datadog-react-native-wizard` (I guess it is either `@` or `/` that is not valid)